modules/components/pages/caches/sql.adoc

```diff
@@ -138,6 +136,8 @@ dsn: foouser:foopassword@tcp(localhost:3306)/foodb
 dsn: postgres://foouser:foopass@localhost:5432/foodb?sslmode=disable
 
 dsn: oracle://foouser:foopass@localhost:1521/service_name
+
+dsn: spanner://projects/fooproject/instances/fooinstance/databases/foodatabase
```

Comment:
I've added this example. Is it correct. Do you think it's worth adding to the other updated components?